### PR TITLE
Language editing

### PR DESCRIPTION
**Pull request by SarahNoack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,15 +24,15 @@ The following images are published on [hub.docker.com](https://hub.docker.com/se
...
-| Container Structure Test|[Container Structure Test](https://github.com/GoogleContainerTools/container-structure-test) with shell to work with `ppiper/jenkins-master`|[ppiper/container-structure-test](https://hub.docker.com/r/ppiper/container-structure-test)|
-| Node Browsers | Web browsers to be used for end to end tests of web applications in Jenkins pipelines | [node-browsers/](https://hub.docker.com/r/ppiper/node-browsers/) |
+| CloudFoundry CLI | Use command line tools for Cloud Foundry with plugins for blue-green deploy and MTA. | [ppiper/cf-cli](https://hub.docker.com/r/ppiper/cf-cli) |
+| Neo CLI | Use SAP Cloud Platform tools for Neo. | [neo-cli/](https://hub.docker.com/r/ppiper/neo-cli/) |
+| Jenkinsfile Runner| Run a `Jenkinsfile` without a long-running, stateful Jenkins master. The [Jenkinsfile Runner](https://github.com/jenkinsci/jenkinsfile-runner) is based on `ppiper/jenkins-master`. | [ppiper/jenkinsfile-runner](https://hub.docker.com/r/ppiper/jenkinsfile-runner) |
+| Life Cycle Container| Use a sidecar image for life-cycle management of the cx-server|[ppiper/cx-server-companion](https://hub.docker.com/r/ppiper/cx-server-companion).|
```
Comment (fwilhe) [inline README.md]: This line should be rephrased as the imperative "use a.." might be confusing here. What we want to convey: This image is a helper thing that facilitates life cycle mgmt for the cx-server.

File: README.md [hunk 2]
```diff
@@ -24,15 +24,15 @@ The following images are published on [hub.docker.com](https://hub.docker.com/se
...
-| Neo CLI | SAP Cloud Platform Tools for Neo. | [neo-cli/](https://hub.docker.com/r/ppiper/neo-cli/) |
-| Jenkinsfile Runner| [Jenkinsfile Runner](https://github.com/jenkinsci/jenkinsfile-runner) based on `ppiper/jenkins-master`, allows running a `Jenkinsfile` without a long-running, stateful Jenkins master. | [ppiper/jenkinsfile-runner](https://hub.docker.com/r/ppiper/jenkinsfile-runner) |
-| Life Cycle Container| Sidecar image for life-cycle management of the cx-server|[ppiper/cx-server-companion](https://hub.docker.com/r/ppiper/cx-server-companion)|
-| Container Structure Test|[Container Structure Test](https://github.com/GoogleContainerTools/container-structure-test) with shell to work with `ppiper/jenkins-master`|[ppiper/container-structure-test](https://hub.docker.com/r/ppiper/container-structure-test)|
-| Node Browsers | Web browsers to be used for end to end tests of web applications in Jenkins pipelines | [node-browsers/](https://hub.docker.com/r/ppiper/node-browsers/) |
+| CloudFoundry CLI | Use command line tools for Cloud Foundry with plugins for blue-green deploy and MTA. | [ppiper/cf-cli](https://hub.docker.com/r/ppiper/cf-cli) |
```
Comment (1084565) [inline README.md]:
```suggestion
| CloudFoundry CLI | Use command line tools for Cloud Foundry with plugins for blue-green deployment and MTA. | [ppiper/cf-cli](https://hub.docker.com/r/ppiper/cf-cli) |
```

File: README.md (hunk 1 above)
Comment (1084565) [reply README.md]: Agreed, that's a valid point. Suggestion:
```suggestion
| Life Cycle Container| Improve life-cycle management of the cx-server by using a sidecar image|[ppiper/cx-server-companion](https://hub.docker.com/r/ppiper/cx-server-companion).|
```

**Comment by SarahNoack** [comment]
@1084565 Could you please check again?

**Comment by 1084565** [comment]
ping @fwilhe

**Pull request merged**